app/models/ability.rb

```diff
@@ -0,0 +1,32 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the user here. For example:
+    #
+    return unless user.present?
+
+    can :read, :all
+    return unless user.admin?
```

Comment:
- [ ] There is no method called `admin?` in your User model. Kindly add that in your User model. Here is example of code for the method.

  ```
  def admin?
    role == 'admin'
  end
  ```